docs/dev/system-design/localvc-localci/localvc-localci-system-design.rst

```diff
@@ -0,0 +1,110 @@
+Artemis supports an integrated version control (VC) and continuous integration (CI) system.
+If you use this *local VC* and *local CI*, the architecture differs from the architecture with external VC and CI systems.
+The deployment with local VC and local CI (without using an external user management system) looks like this:
+
+.. figure:: system-design/localvc-localci/LocalVC_LocalCI_Deployment.png
+   :align: center
+   :width: 800
+   :alt: Local VC and Local CI Deployment
+
+   Local VC and Local CI Deployment
+
+Employing the local VCS and local CIS, administrators and developers can set the Artemis application up without the need for dedicated VCS and CIS installations.
```

Comment:
I would use a lowercase s so that it's clear that it is meant as a plural form. Also, for the "installations", the plural is already in that word
```suggestion
Employing the local VCs and local CIs, administrators and developers can set the Artemis application up without the need for dedicated VC and CI installations.
```

Edit: Is it plural form? I found it used elsewhere where it didn't seem like it 🤔

Reply:
VCS = Version Control System
CIS = Continuous Integration System